File: Java/Hussain Sir Java/Fundamentals/Language/Inheritance/AbstractClassTest.java
import edu.met.banking.*;

class AbstractClassTest{

	public static void main(String[] args){
		Account jack = Banker.openCurrentAccount();
		jack.deposit(15000);
		Account jill = Banker.openSavingsAccount();
		jill.deposit(10000);
		try{
			double payment = Double.parseDouble(args[0]);
			jill.transfer(payment, jack);
		}catch(InsufficientFundsException e){
			System.out.println("Transfer failed due to lack of funds!");
		}catch(Exception e){
			System.out.printf("ERROR: %s%n", e);
		}
		System.out.printf("Jack's Account ID is %d and Balance is %.2f%n", jack.getId(), jack.getBalance());
		System.out.printf("Jill's Account ID is %d and Balance is %.2f%n", jill.getId(), jill.getBalance());
	}
}

File: Fundamentals/Language/Inheritance/abstractclasstest.cs
using Banking;
using System;

static class Program
{
	public static void Main(string[] args)
	{
		Account jack = Banker.OpenCurrentAccount();
		jack.Deposit(20000);
		Account jill = Banker.OpenSavingsAccount();
		jill.Deposit(15000);

		try
		{
			double payment = double.Parse(args[0]);
			jill.Transfer(payment, jack);
			Console.WriteLine("Payment transferred.");
		}
		catch(InsufficientFundsException)
		{
			Console.WriteLine("Transfer failed due to lack of funds!");
		}
		catch(Exception ex)
		{
			Console.WriteLine(ex.Message);
		}

		Console.WriteLine($"Jack's Account ID is {jack.Id} and Balance is {jack.Balance}"); 
		Console.WriteLine($"Jill's Account ID is {jill.Id} and Balance is {jill.Balance}");
	}
}

